privacyscore/backend/management/commands/scanfromfile.py

```diff
@@ -27,24 +27,28 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('file_path')
+        parser.add_argument('scan_list')
         parser.add_argument('-s', '--sleep-between-scans', type=float, default=0)
 
     def handle(self, *args, **options):
         if not os.path.isfile(options['file_path']):
             raise ValueError('file does not exist!')
         self.stdout.write('Reading from file {}'.format(options['file_path']))
 
-        start = timezone.now()
+        scan_list = ScanList.objects.get_or_create(name=options['scan_list'])[0]
+        scan_list.private = True
+        scan_list.save()
 
         with open(options['file_path'], 'r') as fdes:
-            # TODO: use bulk create for db optimization?
-            sites = {
-                Site.objects.get_or_create(
-                    url=normalize_url(url))[0]
-                for url in fdes.readlines()
-                if '.' in url
-            }
+            sites = []
+            for url in fdes.readlines():
```

Comment:
I'd probably do smth like `(u.strip() for u in fdes)` or so in order to be more lenient when accepting data.
What do you think about making that change?